portality/formcontext/formcontext.py

```diff
@@ -582,7 +618,13 @@ def finalise(self):
             # create the user account for the owner and send the notification email
             try:
                 owner = self._create_account_on_suggestion_approval(self.target, j)
-                self._send_application_approved_email(j.bibjson().title, owner.name, owner.email, self.source.current_journal is not None)
+                journal_contacts = ""
+                for contact in j.contacts():
+                    names = contact.get("name")
+                    journal_contacts = ", ".join(names)
```

Comment:
Do you mean `journal_contacts` += `", ".join(names)`? It appears here that the latest contact's name will be the only one shown, as the variable is being overwritten in this loop. So the last iteration of the loop will be all that remains.